packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -215,29 +239,29 @@ const Dropdown = ({
         const focusedElement = document.activeElement;
         switch (key) {
           case 'Enter':
-            const match = focusedElement && focusedElement.id.match(`${state.id}-option-(.*)`);
+            const match = focusedElement && focusedElement.id.match(`${name}-option-(.*)`);
             if (match) {
-              handleSelect(match[1]);
+              handleSelect(match[1]); // TODO: fix this
```

Comment:
Can't tell if this comment is a remnant from development or just needs to be more descriptive.

Reply:
These are things I still plan on fixing before un-drafting the PR :)

Reply:
More like notes for you than me haha